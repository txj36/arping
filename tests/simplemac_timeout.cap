IP 192.0.2.100 > 255.255.255.255: ICMP echo request, id 27006, seq 0, length 28
	0x0000:  4500 0030 697e 0000 4001 4eeb c000 0264
	0x0010:  ffff ffff 0800 113f 697e 0000 06f7 0000
	0x0020:  0000 0000 b25a f716 0000 0000 fabf d219
