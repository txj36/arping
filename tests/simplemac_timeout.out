ARPING 00:01:02:33:44:01
Timeout

--- 00:01:02:33:44:01 statistics ---
1 packets transmitted, 0 packets received, 100% unanswered \(0 extra\)
